client/src/controllers/RevealController.ts

```diff
@@ -214,6 +217,37 @@ export class RevealController extends Controller<HTMLElement> {
       });
   }
 
+  get stored() {
+    const storeValue = this.storeValue;
+    const key = `wagtail:${this.identifier}:opened`;
+    if (storeValue) {
+      try {
+        const storedValue = localStorage.getItem(key);
+        if (storedValue === storeValue) {
+          return true;
+        }
+      } catch (error) {
+        // Ignore if localStorage is not available
+      }
+    }
+    return false;
+  }
+
+  set stored(isOpened) {
+    const storeValue = this.storeValue;
+    const key = `wagtail:${this.identifier}:opened`;
+
+    try {
+      if (isOpened) {
+        localStorage.setItem(key, storeValue);
+      } else {
+        localStorage.removeItem(key);
```

Comment:
It looks like we'll need a unit test for this branch, when closing the breadcrumbs.